.idea/dictionaries/brianwaganer.xml

```diff
@@ -0,0 +1,8 @@
+<component name="ProjectDictionaryState">
```

Comment:
.idea/dictionaries/brianwaganer.xml should not be in your PR we don't want that merged to the kb repo